tools/sgterraform/tools.go

```diff
@@ -21,6 +22,30 @@ func Command(ctx context.Context, args ...string) *exec.Cmd {
 	return sg.Command(ctx, sg.FromBinDir(binaryName), args...)
 }
 
+func CommentPRWithTFPlan(ctx context.Context, prNumber, environment, tfPlan string) *exec.Cmd {
```

Comment:
Can remove TF to avoid repetition - how about `CommentOnPullRequestWithPlan`?

Reply:
Yes!